src/components/Footer/Footer.js

```diff
@@ -0,0 +1,5 @@
+const Footer = (props) =>  (
+    <div className={`text-center m-5-${props.theme}`}>Wypożyczalnia FreeRide 2021</div>
+);

```

Comment:
Please use dynamic date something like new Date().getFullYear()

Reply:
You haven't specified PropTypes for this and some others components

Reply:
Ok, tomorrow I should change date 


yes, i know about props, later i do sth with this